linode/manager
# upcoming: [M3-8270] - Add Distributed Icon to ImageSelects for distributed compatible images

## Description 📝

- Adds a Distributed Icon to Images in the ImageSelect on the Linode Create flow that are distributed region capable 🌐 

## Preview 📷

![Screenshot 2024-06-18 at 11 45 40 AM](https://github.com/linode/manager/assets/115251059/dcb90ac8-8678-4605-ab65-6991ef28c65f)

## How to test 🧪

### Prerequisites
- Turn on the MSW 💡 

### Verification steps
- Verify  you see a distributed icon for the image labeled `distributed-image` in the ImageSelect on the Linode Create flow 👁️ 
  - Test both Linode Create v1 and v2 🚨

## As an Author I have considered 🤔
